kiva/agg/src/graphics_context.i

```diff
@@ -722,11 +722,27 @@ namespace kiva {
 
             void clear_clip_path();
             void clear(agg24::rgba& value=_clear_color);
+
+            // Exception handler for cell block overflow
+            %exception {
+                try
+                {
+                    $action
+                }
+                catch (const std::overflow_error& exn)
+                {
+                    PyErr_SetString(PyExc_OverflowError, exn.what());
+                    return NULL;
+                }
+            }
+
             void stroke_path();
             void fill_path();
             void eof_fill_path();
             void draw_path(draw_mode_e mode=FILL_STROKE);
```

Comment:
Correct me if i'm wrong but exceptions in these 4 calls are handled by the surrounding `%exception` definitions?

Reply:
That's my understanding